plugins/inputs/sqlserver/connectionstring.go

```diff
@@ -0,0 +1,100 @@
+package sqlserver
+
+import (
+	"net/url"
+	"strings"
+)
+
+const (
+	emptyServerName   = "<empty-server-name>"
+	emptyDatabaseName = "<empty-database-name>"
+)
+
+// getConnectionIdentifiers returns the sqlInstance and databaseName from the given connection string.
+// The name of the server is returned as-is in the connection string
+// If the connection string could not be parsed or sqlInstance/databaseName were not present, a placeholder value is returned
+func getConnectionIdentifiers(connectionString string) (sqlInstance string, databaseName string) {
+	if len(connectionString) == 0 {
+		return "", ""
```

Comment:
IIRC you want `emptyServerName` and `emptyDatabaseName` here!?!? :-)

Reply:
Good catch.